### ISSUE TITLE
@graphql-eslint/unique-operation-name outputting redundant directories 

### ISSUE THREAD

**Issue by jaylucas** [dotansimha/graphql-eslint#376]
**Describe the bug**
For some reason I get circular checks against the corresponding .graphql file 

```
/Users/<USERNAME>/workspace/GqlSchema/operations/getLocations.graphql
  1:7  error  Operation named "GetLocations" already defined in:
        /Users/<USERNAME>/workspace/GqlSchema/Users/<USERNAME>/workspace/GqlSchema/operations/getLocations.graphql
```
**To Reproduce**

```
{  
    "root": true,
    "overrides": [
      {
        "files": ["*.graphql"],
        "parser": "@graphql-eslint/eslint-plugin",
        "parserOptions": {
            "operations": ["./operations/**/*.graphql"]
        },
        "plugins": ["@graphql-eslint"],
        "rules": {
          "@graphql-eslint/unique-operation-name": "error"
        }
      }
    ]
  }
```

```
  "scripts": {
    "clean": "rm -f schema.graphql index.d.ts Types.kt",
    "codegen": "npm run graphql-codegen",
    "lint": "eslint --config .eslintrc.json --ext .graphql .",
  },
  "devDependencies": {
    "@graphql-codegen/cli": "^1.19.0",
    "@graphql-codegen/schema-ast": "^1.17.8",
    "@graphql-codegen/typescript": "^1.17.11",
    "@graphql-codegen/typescript-operations": "^1.17.8",
    "@graphql-codegen/kotlin": "^1.17.10",
    "@graphql-eslint/eslint-plugin": "0.9.1",
    "eslint": "7.14.0",
    "graphql": "^15.4.0",
    "prettier": "2.2.1"
  }
```

**Expected behavior**
My assumption is that this wouldn't fail.

**Environment:**

- OS: OSX 10.15.7
- `@graphql-eslint/...`: 0.9.1
- NodeJS: 12.8

**Comment by dotansimha** [comment]
Fixed, we'll cut a release very soon!

**Comment by dotansimha** [comment]
Fixed in `v1.0.0`! 🎉
